r2-testapp/src/main/java/org/readium/r2/testapp/db/Database.kt

```diff
@@ -0,0 +1,43 @@
+package org.readium.r2.testapp.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import org.readium.r2.testapp.domain.model.Book
+import org.readium.r2.testapp.domain.model.Bookmark
+import org.readium.r2.testapp.domain.model.Highlight
+import org.readium.r2.testapp.domain.model.OPDS
+
+@Database(
+    entities = [Book::class, Bookmark::class, Highlight::class, OPDS::class],
+    version = 1,
+    exportSchema = false
+)
+abstract class BookDatabase : RoomDatabase() {
```

Comment:
Much nicer to have a single database 👍 

```suggestion
abstract class Database : RoomDatabase() {
```